package/kernel/linux/modules/hwmon.mk

```diff
@@ -93,6 +93,20 @@ endef
 $(eval $(call KernelPackage,hwmon-ina2xx))
 
 
+define KernelPackage/hwmon-it87
+  TITLE:=it87 monitoring support
```

Comment:
Done

On 16-05-18 04:45 AM, Stijn Tintel wrote:
> In package/kernel/linux/modules/hwmon.mk
> <https://github.com/lede-project/source/pull/26#discussion_r63666330>:
> 
>> @@ -93,6 +93,20 @@ endef
>>  $(eval $(call KernelPackage,hwmon-ina2xx))
>>  
>>  
>> +define KernelPackage/hwmon-it87
>> +  TITLE:=it87 monitoring support
> 
> Minor nit: other modules use uppercase characters before the number, I
> would prefer the same here for consistency
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/lede-project/source/pull/26/files/ce4544b784b23e104fa92503403d39030f0f735b#r63666330>
>